packages/transformer/src/IModelTransformer.ts

```diff
@@ -1571,9 +1585,69 @@ export class IModelTransformer extends IModelExportHandler {
         targetElementProps.jsonProperties.Subject.Job = undefined;
       }
     }
+
+    if (
+      this.ecefTransform !== undefined &&
+      sourceElement instanceof GeometricElement3d
+    ) {
+      // can check the sourceElement since this IModelTransformer does not remap classes
+      const placement = Placement3d.fromJSON(
+        (targetElementProps as GeometricElement3dProps).placement
+      );
+
+      if (placement.isValid) {
+        placement.multiplyTransform(this.ecefTransform);
+        (targetElementProps as GeometricElement3dProps).placement = placement;
+      }
+    }
     return targetElementProps;
   }
 
+  /**
+   * Calculate the transform between two ECEF locations
+   * @param srcDb
+   * @param targetDb
+   * @returns Transform that converts relative coordinates in the source iModel to relative coordinates in the target iModel.
+   * @note This can only be used if both source and target iModels are linearly located
+   */
+  public getEcefTransform(srcDb: IModelDb, targetDb: IModelDb): Transform {
```

Comment:
nit: the `get` prefix keeps bothering me because it feels like you're getting something from the Transformer, rather than computing something from two iModels... `calculateEcefTransform`? Please disregard if you hate this.